Instastories.py

```diff
@@ -203,23 +211,53 @@ def nicks_to_ids(usr_list):
     return ids    
 
 
+def numberOfPeople():
+
+    count = int(input("Please insert the number of people to scrape: "))
+    return count
+
+
+
+def logToHTML():
+    rendered_log = []
+    if not os.path.exists("run_history.log"):
+        f = open("run_history.log", "w").close()
+
+    with open("run_history.log", "r") as o:
+        for i in o.readlines():
+            rendered_log.append(str(i))
+        return rendered_log
+
+
+
+
+def startScrape(cookie, amountScraped):
+    stories = get_stories_tray(cookie)                                        
+    ids = tray_to_ids(stories)                                                
+    count_i, count_v = download_today_stories(ids , cookie, amountScraped) 
+
+    timestampStr = datetime.datetime.now().strftime("%d-%b-%Y (%H:%M:%S)")
+
+    with open("run_history.log", "a+") as o:
```

Comment:
This depends that you did used log_to_html before, otherwise the file might not exists. Change the mode so it create it if it's missing